core/src/com/unciv/ui/victoryscreen/VictoryScreen.kt

```diff
@@ -175,22 +178,84 @@ class VictoryScreen(val worldScreen: WorldScreen) : PickerScreen() {
 
     private fun setCivRankingsTable() {
         val majorCivs = gameInfo.civilizations.filter { it.isMajorCiv() }
-        val civRankingsTable = Table().apply { defaults().pad(5f) }
+
+        contentsTable.clear()
+        
+        if (UncivGame.Current.settings.useDemographics) contentsTable.add(buildDemographicsTable(majorCivs))
+        else contentsTable.add(buildRankingsTable(majorCivs))
+    }
+
+    enum class RankLabels { Demographic, Rank, Value, Best, Average, Worst}
+    private fun buildDemographicsTable(majorCivs: List<CivilizationInfo>): Table {
+        val demographicsTable = Table().apply { defaults().pad(5f) }
+        for (rankLabel in RankLabels.values())   {
+            if (rankLabel == RankLabels.Demographic) {
```

Comment:
The indent level of this if could be saved - rankLabel == RankLabels.Demographic is the static header row, and by pushing the header row block outside the loop and having the loop start one later (`in RankLabels.values().drop(1)` maybe?) you could do without the if? Besides that, the function already looks near fine readability-wise as is.

Reply:
I'd personally just remove `Demographic` from the entire Enum and just handle that column seperately. It's very different from all other columns anyway

Reply:
I agree with xlen on this :)

Reply:
Demographic removed from the enum and its static header row moved to new function buildDemographicsHeaders.